pkg/runner/runner_test.go

```diff
@@ -83,10 +83,14 @@ func TestRunEvent(t *testing.T) {
 	platforms := map[string]string{
 		"ubuntu-latest": "node:12.20.1-buster-slim",
 	}
+	pwshplatforms := map[string]string{
+		"ubuntu-latest": "ghcr.io/justingrote/act-pwsh:latest",
+	}
 	tables := []TestJobFileInfo{
 		{"testdata", "basic", "push", "", platforms, ""},
 		{"testdata", "fail", "push", "exit with `FAILURE`: 1", platforms, ""},
 		{"testdata", "runs-on", "push", "", platforms, ""},
+		{"testdata", "powershell", "push", "", pwshplatforms, ""},
```

Comment:
perhaps it would be better as inline map instead of single variable

Reply:
@catthehacker I did it expecting more tests to utilize it, but I can make it an inline map if you prefer.

Reply:
@catthehacker I did it expecting more tests to utilize it, but I can make it an inline map if you prefer.

Reply:
Fixed in https://github.com/nektos/act/pull/660/commits/a91679c83c55128751c14c6c5bbb6bdaf02c413c